src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  box-sizing: border-box;
+  padding: 50px 40px;
+  display: grid;
+  place-content: center;
+  place-items: center;
+  row-gap: 48px 46px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  transition: all 300ms ease;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 0;
+  }
+
+  &__item {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+
+    background-size: contain;
+    background-image: url(../images/imac.jpeg);
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    margin-top: 4px;
+    margin-bottom: 0;
+    color: $grey-text-color;
+  }
+
+  &__price {
+    position: relative;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $grey-text-color;
+  }
+
+  &__price::after {
+    position: absolute;
+    content: '$2,199';
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $main-text-color;
+    right: 0;
+  }
+
+  &__buy {
+    display: flex;
+    margin: 16px 0;
+  }
+
+  &__button {
+    color: $white-color;
+    background-color: $blue-color;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+    text-decoration: none;
+    line-height: 40px;
+    height: 40px;
+    width: 166px;
+    border-radius: 5px;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
```

Comment:
The hover effect increases the size of the `.card` element, which could affect the layout of surrounding elements. Ensure that this scaling does not cause layout shifts or overlap issues with other content.